RIPS/rip-7712.md

```diff
@@ -39,26 +39,43 @@ with ERC-4337, as all ERC-4337 Smart Contract Accounts already use the solution
 
 ## Specification
 
+### Constants
+
+| Name                     | Value |
+|--------------------------|-------|
+| AA_BASE_GAS_COST         | 15000 |
+| AA_BASE_GAS_COST_RIP7712 | 10000 |
+
 ### Non-sequential nonce support
 
-We propose an introduction of a separate 2-dimensional `nonce` value used when contracts initiate a transaction.
```

Comment:
If nonceKey is zero (non-existent), then the current system nonce value is in use, and the transaction's base cost is `AA_BASE_GAS_COST`.
If nonceKey is non-zero, then the current system nonce is not in use for validation (and also is not incremented), and the base cost is `AA_BASE_GAS_COST_RIP7712`, as the validation gas cost will include the cost of calling the nonce manager.

During transaction creation, the nonceKey MUST be zero.